components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -1,16 +1,235 @@
+      description: 'Ready to explore Qiskit’s capabilities for yourself? Copy the code to the right of this paragraph and try running it in your local Python environment. You can also click the link to the IBM Quantum Lab and test it there.',
+      thumbnailResource: '/images/landing-page/med_02_1.png'
+    }
+  ]
+
+  quantumLabCTA = {
+    url: 'https://quantum-computing.ibm.com/lab/files/qiskit-textbook/getting-started/example.ipynb',
+    label: 'Open in Quantum Lab',
+    segment: { cta: 'quantum-lab', location: 'homepage-capabilities-section' }
+  }
+
+  copyCodeCTA = {
+    url: '',
+    label: 'Copy code',
+    segment: { cta: 'copy-code', location: 'homepage-capabilities-section' }
+  }
+
+  isActiveSection (item: QiskitCapability, index: number): boolean {
+    if (this.capabilities[this.capabilities.length - 1].title === this.activeSection) {
+      return true
+    }
+
+    return item.title === this.activeSection || (this.activeSection === '' && index === 0)
+  }
+
+  copyToClipboard (): void {
+    const codeSnippet = document.querySelector('.code-cell')?.textContent!
+    navigator.clipboard.writeText(codeSnippet)
+  }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '~carbon-components/scss/globals/scss/typography';
+
+$cta-max-width: 14rem;
+
+.qiskit-capabilities-section {
+  padding-bottom: $spacing-09;
+
+  &__container {
+    display: flex;
+    padding-top: $spacing-05;
+    padding-bottom: $spacing-09;
+
+    @include mq($until: medium) {
+      padding-bottom: $spacing-05;
+    }
+  }
+
+  &__capabilities {
+    display: flex;
+    align-items: flex-start;
+    width: 100%;
+
+    @include mq($until: medium) {
+      flex-direction: column;
+    }
+  }
+
+  &__card {
+    min-height: 12rem;
+    margin-bottom: $spacing-07;
+
+    &:last-child {
+      margin-bottom: 10rem;
+
+      @include mq($until: medium) {
+        margin-bottom: $spacing-07;
+      }
+    }
+  }
+
+  &__scrolling-ui {
+    position: sticky;
+    top: $spacing-06;
+    flex: 0 0 1;
+    max-width: 100%;
+    min-height: 20rem;
+
+    @include mq($from: x-large) {
+      flex: 0 0 38rem;
+    }
+
+    @include mq($from: large, $until: x-large) {
+      flex: 0 0 34rem;
+    }
+
+    @include mq($until: large) {
+      flex: 0 0 22rem;
+    }
+
+    @include mq($until: medium) {
+      flex: initial;
+      width: 100%;
+      margin-bottom: $spacing-07;
+    }
+  }
+
+  &__ctas {
+    display: flex;
+    justify-content: flex-end;
+
+    @include mq($until: medium) {
+      flex-direction: column-reverse;
+    }
+
+    &__link {
+      width: 100%;
+      max-width: $cta-max-width;
+      padding: $spacing-05;
+      margin-right: $spacing-05;
+      justify-content: space-between;
+
+      @include mq($until: medium) {
+        max-width: initial;
+      }
+    }
+  }
+
+  &__copy-button {
+    display: flex;
+    width: 100%;
+    max-width: $cta-max-width;
+    padding-right: $spacing-05;
+    justify-content: space-between;
```

Comment:
I believe this has no noticeable effect, but let me know if I'm missing something.

```suggestion
```